lingua_franca/format.py

```diff
@@ -385,7 +387,8 @@ def nice_duration(duration, lang=None, speech=True):
         str: timespan as a string
     """
     if not lang:
-        warn(NoneLangWarning)
+        if lang == None:
```

Comment:
nitpick, should be ```if lang is None```

https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none

Reply:
aww crud that got missed in a push just now